Screenbox/Pages/SettingsPage.xaml

```diff
@@ -17,47 +18,63 @@
     xmlns:strings="using:Screenbox.Strings"
     xmlns:triggers="using:CommunityToolkit.WinUI"
     xmlns:ui="using:CommunityToolkit.WinUI"
+    d:DesignHeight="1472"
+    d:DesignWidth="848"
     mc:Ignorable="d">
 
     <Page.Resources>
         <ResourceDictionary>
-            <x:Double x:Key="ToggleSwitchPreContentMargin">4</x:Double>
-            <x:Double x:Key="ToggleSwitchPostContentMargin">4</x:Double>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="ms-appx:///CommunityToolkit.WinUI.Controls.SettingsControls/SettingsCard/SettingsCard.xaml" />
```

Comment:
Why do we need to manually import the dictionary here?

Reply:
It's needed for the SettingsCard style

Reply:
The SettingsCard style is not imported automatically?

Reply:
You'd think so, but it won't build without it